docs/kernel_troubleshooting.rst

```diff
@@ -0,0 +1,183 @@
+  older version is only a stopgap measure to avoid a prolonged outage of your
+  SecureDrop instance.
+
+Inspect the file ``/boot/grub/grub.cfg``. You should find a ``menuentry`` line
+with the same text that you selected during boot, e.g.:
+
+.. code:: sh
+
+  submenu 'Advanced options for Ubuntu'…
+
+    menuentry 'Ubuntu, with Linux 4.xxx.xx-grsec…
+
+Take note of its position among the other submenu entries (it will most likely
+be third). Then edit the GRUB configuration:
+
+.. code:: sh
+
+  sudo vim /etc/default/grub
+
+Make a backup of the file or take a note of the current value of
+``GRUB_DEFAULT`` somewhere, so you can restore the previous behavior easily at a
+later point.
+
+Once you have done so, set the ``GRUB_DEFAULT`` variable to point to the index
+of the  menu and submenu. Note that the index starts at 0, so for a typical
+setup, the line in ``/etc/default/grub`` would look like this:
+
+.. code:: sh
+
+  GRUB_DEFAULT=”1>2”
+
+The “1” means the second entry of the main menu (“Advanced options”),
+the “2” means the third entry of the submenu. Again, update these
+numbers consistent with your configuration. 
+
+
+.. caution:: Ensure that you have chosen the right index for the main menu
+  and the submenu, and double-check that you are beginning the count at 0, not
+  1; otherwise, you may boot into the wrong kernel.
+
+This change still has to be applied to take effect on the next boot:
+
+.. code:: sh
+
+  sudo update-grub2
+
+Now you can reboot into the old, working kernel.
+
+.. code:: sh
+
+  sudo reboot
+
+The server should come up automatically. From here on, you should be
+able to perform all administrative tasks via SSH as usual. If you want
+additional confirmation of the kernel version, the command 
+``uname -r`` should display the expected kernel version number.
+
+Please notify us of the compatibility issue so we can help you resolve it ASAP.
+
+Report Compatibility Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have encountered issues with a kernel update, it is important
+that you report them to us so that we may incorporate any necessary
+changes to our updated kernel, and so that we can work with you to
+switch back to the new kernel as soon as possible.
+
+Run the following commands via SSH from the *Admin Workstation*:
+
+.. code:: sh
+
+  cd ~/Persistent/securedrop/
+  source .venv/bin/activate
+  cd install_files/ansible-base
+  ansible all -b -m setup > server-facts.log
+
+Please also send us a copy of ``/var/log/syslog`` and ``/var/log/dmesg`` for
+analysis.
+
+You can share ``server-facts.log``, ``syslog`` and ``dmesg`` with us as follows:
+
+-  If you are a member of our Support Portal, please create a new issue
+   and attach the files to it.
+-  Alternatively, email us at securedrop@freedom.press 
+   (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__) 
+   with the subject “SecureDrop kernel facts” and the files attached.
+
```

Comment:
I suggest adding some language to discourage users from sharing raw logs and sensitive information to public forums such as GitHub or securedrop.club, but rather to share only high-level information.

Reply:
done!